.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+# 📝 Summary
+
+**Ticket Summary (Title):**  
+<!-- Use the exact title from Jira or a brief, clear summary -->
+
+**Ticket Link:**  
+<!-- e.g. https://issues.redhat.com/browse/ACM-12345 -->
+
+**Type of Change:**  
+<!-- Select one -->
+- [ ] 🐞 Bug Fix  
+- [ ] ✨ Feature  
+- [ ] 🔧 Refactor
+- [ ] 💸 Tech Debt
+- [ ] 🧪 Test-related  
+- [ ] 📄 Docs
+
+---
+
+## ✅ Checklist
+
+### General
+
+- [ ] PR title follows the convention (e.g. `[ACM-1234] Fix bug with...`)
```

Comment:
Are the brackets around ACM-1234 literal? Also, we are up to 5 digits in our bug numbers now.